harness/src/main/java/org/jboss/pvt/harness/utils/HttpUtils.java

```diff
@@ -26,12 +28,18 @@
  */
 public class HttpUtils {
 
-    public static void httpDownload(String httpUrl) throws Exception {
+    private static Logger logger = LoggerFactory.getLogger(HttpUtils.class);
+
+
+    public static File httpDownload(String httpUrl) throws Exception {
         String filename = httpUrl.substring(httpUrl.lastIndexOf("/") + 1);
-        httpDownload(httpUrl, filename);
+        return httpDownload(httpUrl, filename);
     }
 
-    public static void httpDownload(String httpUrl,String saveFile) throws Exception {
-        FileUtils.copyURLToFile(new URL(httpUrl), new File(saveFile));
+    public static File httpDownload(String httpUrl,String saveFile) throws Exception {
+        File savedFile = new File(saveFile);
+        logger.info("Downloading " + httpUrl);
```

Comment:
Use , separation on logging to avoid string concatenation if that logging level is disabled